app/main/templates/index.html

```diff
@@ -3,117 +3,86 @@
 
 {% block app_content %}
 
-<style>
+    <div class="container pt-5">
 
-    h1 {
-      text-align: center;
-      padding: 64px;
-      background: linear-gradient(to right, hsl(0, 0%, 30%) 0, hsl(0, 0%, 100%) 10%, hsl(0, 0%, 30%) 20%);
```

Comment:
I'm not quite clear where these have moved to or how they were overwritten

Reply:
It was only being used on the old front page. It was basically trying to apply styling to all Heading 1s, but `hazen.css` now takes care of this (in combination with Bootstrap). Also, it's better practice not to have CSS snippets dotted around your .html files. It should all be controlled in your CSS files.